devolutions-gateway/Cargo.toml

```diff
@@ -94,6 +94,7 @@ packet = { git = "https://github.com/fdubois1/rust-packet.git" }
 
 # For KDC proxy
 portpicker = "0.1"
+network-scanner = { version = "0.0.0", path = "../crates/network-scanner" }
```

Comment:
nitpick: This is unrelated to the KDC proxy, you can move this to the "in-house" section instead

Reply:
updated